src/components/client/campaigns/CampaignDetails.tsx

```diff
@@ -53,11 +53,13 @@ const StyledGrid = styled(Grid)(({ theme }) => ({
 
     [theme.breakpoints.up('xs')]: {
```

Comment:
This breakpoint seems redundant to me.  Can you try, setting the `fontSize` and `marginBottom` properties without it, and see if it works? 
The text should still be 32px on viewports smaller than 900px, and 45px on viewports bigger than 900px.